src/main/java/com/vinylteam/vinyl/entity/UserPost.java

```diff
@@ -0,0 +1,101 @@
+package com.vinylteam.vinyl.entity;
+
+import java.util.Objects;
+
+public class UserPost {
+    private long id;
+    private long userId;
+    private String name;
+    private String email;
+    private String theme;
+    private String message;
+
+    public UserPost() {
+    }
+
+    public UserPost(long userId, String name, String email, String theme, String message) {
+        this.userId = userId;
+        this.name = name;
+        this.email = email;
+        this.theme = theme;
+        this.message = message;
+    }
```

Comment:
Deep copy constructor is unnecessary unless you are using it in the src/main/... code.